mypy.ini

```diff
@@ -1,11 +0,0 @@
-[mypy]
-warn_return_any=True
-warn_unused_configs=True
-strict=True
-namespace_packages=True
-
-[mypy-nox.*]
-ignore_missing_imports=True
```

Comment:
What happened to `ignore-missing-imports` for the nox mypy module?

Reply:
It wasn't doing anything. Nox is typed so im not sure why that was there to begin with.

Reply:
CI passes so it clearly has types